firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandler.java

```diff
@@ -319,6 +321,12 @@ private FetchResponse fetchFromBackend(
               getFirstOpenTime(),
               currentTime);
 
+      if (response.getFetchedConfigs() != null) {
+        // Set template version in memory.
```

Comment:
nit: prob don't need to say memory to refer to the instance variable. that threw me off at first

Reply:
Done